conf/application.conf

```diff
@@ -164,3 +164,5 @@ proxy {
 auth {
   api-token = "9253947-99f3-47d7-9af2-b75b4f37fd34"
 }
+
+caseReferenceStart = 504400000
```

Comment:
Very minor but I think we generally go with the format `case-reference-start` in the config files

Reply:
Done. 

Stefano did talk about that name but talked ourselves into camel casing it :)